### ISSUE TITLE
SRTP protect error on Mac OS

### ISSUE THREAD

**Issue by adrianbg** [meetecho/janus-gateway#703]
Janus version: 6ab0979562155ea9db2ea0df619df144fd8063fe (also with v0.2.0)

Built and running on Mac OS 10.11.6

When I browse to localhost:8000/audiobridgetest.html in Chrome and click "Start" I get the following. The same thing happens when I use OpenWebRTC or AppRTC from iOS. All of the above work fine when I build/run Janus in an Ubuntu VM on the same  machine.

[Full log](https://github.com/meetecho/janus-gateway/files/637198/srtp_protect.txt)

Log excerpt:

```
[transports/janus_http.c:janus_http_handler:1176] Done getting payload, we can answer
{"janus":"trickle","candidate":{"completed":true},"transaction":"kCWv7gEceGmt"}
Forwarding request to the core (0x7fb72b417630)
Got a Janus API request from janus.transport.http (0x7fb72b417630)
Transport task pool, serving request
No more remote candidates for handle 8406644381804247!
Sending Janus API response to janus.transport.http (0x7fb72b417630)
Got a Janus API response to send (0x7fb72b417630)
[transports/janus_http.c:janus_http_request_completed:1667] Request completed, freeing data
[ERR] [ice.c:janus_ice_send_thread:3486] [8406644381804247] ... SRTP protect error... err_status_bad_param (len=19-->19, ts=960, seq=1)...
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -4 (corrupted stream)
[8406644381804247] Notifying that we are receiving audio
[8406644381804247] Sending event to transport...
Sending event to janus.transport.http (0x7fb72b411590)
Got a Janus API event to send (0x7fb72b411590)
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -1 (invalid argument)
Still prebuffering (got 1 packets), not adding the user to the mix yet
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -4 (corrupted stream)
[ERR] [ice.c:janus_ice_send_thread:3486] [8406644381804247] ... SRTP protect error... err_status_bad_param (len=19-->19, ts=3840, seq=4)...
[8406644381804247]  DTLS already set up, disabling retransmission timer!
[WARN] [8406644381804247] Not video and not audio? dropping (SSRC 2147483649)...
[ERR] [ice.c:janus_ice_send_thread:3486] [8406644381804247] ... SRTP protect error... err_status_bad_param (len=19-->19, ts=4800, seq=5)...
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -4 (corrupted stream)
[WARN] [8406644381804247] Not video and not audio? dropping (SSRC 2858151084)...
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -4 (corrupted stream)
Still prebuffering (got 2 packets), not adding the user to the mix yet
[ERR] [ice.c:janus_ice_send_thread:3486] [8406644381804247] ... SRTP protect error... err_status_bad_param (len=19-->19, ts=7680, seq=8)...
[ERR] [plugins/janus_audiobridge.c:janus_audiobridge_incoming_rtp:1959] [Opus] Ops! got an error decoding the Opus frame: -4 (corrupted stream)
```

**Comment by lminiero** [comment]
I'm wondering if the `err_status_bad_param` comes from the RTP extensions support we added to the AudioBridge in #697. Try adding `audiolevel_ext=no` to the configuration of the room you're using and restart Janus (or pass a `audiolevel_ext:false` as an additional parameter when creating a room via API) and see if the SRTP protect error is still there.

**Comment by adrianbg** [comment]
No dice -- it's still there. Any other ideas?

**Comment by adrianbg** [comment]
Oh btw, I'd also tested with v0.2.0 so this must've been introduced before then. Maybe I'll try some earlier versions.

**Comment by adrianbg** [comment]
Also affects v0.1.2 except the Opus decoding errors don't appear. Think this is related to some library I have on my system?

**Comment by lminiero** [comment]
> No dice -- it's still there. Any other ideas?

Yeah I thought it wouldn't -- thinking about it that patch only makes the plugin able to process extensions, but doesn't generate any of its own, so that couldn't be the cause.

Not sure about the cause as my familiarity wit Mac OS and its libraries is pretty much zero. If you can narrow down which commit starts giving you the Opus decoding errors it might help figure out if it's us doing something wrong somewhere.

**Comment by lminiero** [comment]
This should have been fixed by #707 which we merged, feel free to reopen if that's not the case.
